src/postprocess.cpp

```diff
@@ -1,27 +1,41 @@
 
+#include "prints.cpp"
+#include "utils.cpp"
 #include <algorithm>
 #include <array>
 #include <math.h>
 #include <numeric>
 // #include <pybind11/pybind11.h>
 // #include <pybind11/stl.h>
 #include <vector>
-// namespace py = pybind11;
-#include "prints.cpp"
-#include "utils.cpp"
 
+// namespace py = pybind11;
 
 class NMS {
-
-  // NMS() {
-
-  // }
+private:
+  float nms_thres;
+  float conf_threshold = 0.2;
+  bool refine_vis = false;
+  float vis_thresh = 0.5;
+  AnchorMat proposals_arr;
+  bool is_test;
 
 public:
+  // NMS(py::list &proposals) { proposals_arr =
+  // convert_pylist_to_arr(proposals); }
+  NMS(AnchorMat proposals, int nms_thold, bool is_test_flag) {
```

Comment:
No .hpp files. Each cpp file must have a corresponding .hpp files which defines the structure of the file. 
The purpose of .hpp is to give a high level overview of the functions and the classes by declarations and the doxygen comments go there.
A .cpp file includes the defnintion and implementation of the corresponding functions and classes and adds more details if needed using docygen tags

Reply:
True. I have added the corresponding .hpp files